portfolio/src/main/webapp/script.js

```diff
@@ -29,8 +29,48 @@ hideables.forEach(hideable => {
   h3.addEventListener('click', () => ul.classList.toggle("hidden"));
 });
 
-async function getGreeting() {
-    const response = await fetch("/data");
-    const quote = await response.text();
-    document.getElementById("quote-container").innerHTML = quote;
-}
\ No newline at end of file
+async function getComments() {
+  let formNumber = document.getElementById("number-comments").value;
+
+  const response = await fetch(`/data?number=${formNumber}`);
+  const text = await response.text();
+  const comments = JSON.parse(text);
+  
+  const fatherDiv = document.getElementById("comments-container");
```

Comment:
try to use more expressive variable names, i.e. `commentsContainer` instead of `fatherDiv`.